client/src/reviewList.jsx

```diff
@@ -1,47 +1,57 @@
 import React from 'react';
-import {Grid, Image, Divider} from 'semantic-ui-react';
+var moment = require('moment');
```

Comment:
are you using this instead of semantic UI? curious to hear the reason for the change

Reply:
No, I'm not using semantic UI at all right now. I'm using moment to format my date though